### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by wmantly**
* Bumped version of DSL maker and fpm
* Dry run works

### CODE REVIEW

File: packager.gemspec [hunk 1]
```diff
@@ -22,9 +22,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_dependency 'dsl_maker', '~> 0.1', '>= 0.1.1'
-  s.add_dependency 'fpm', '~> 1.4', '>= 1.1.0'
-  s.add_dependency 'thor', '~> 0.0', '>= 0.19.0'
+  s.add_dependency 'dsl_maker', '>= 0.1.1'
```
Comment (robkinyon) [inline packager.gemspec]: This needs to be `'~> 1.0', '>= 1.0.0'` in order to force the usage of the version with the MIT license.

File: lib/packager/executor.rb [hunk 2]
```diff
@@ -52,8 +52,11 @@ def create_package_for(item)
     end
 
     def execute_command(cmd)
-      return if dryrun
-
+      if dryrun
+        puts 'Dry run! This has not run or changed anything'
+        puts "#{cmd.to_system.join(' ')}"
+      end
```
Comment (robkinyon) [inline lib/packager/executor.rb]: This needs to have a `return` in this block in order to avoid the invocation of the command.

Comment (wmantly) [reply lib/packager/executor.rb]: Good catch!

File: packager.gemspec (hunk 1 above)
Comment (wmantly) [reply packager.gemspec]: Should this be changed to MIT, its currently GPL2?

**Pull request closed without merging**